cura/CuraApplication.py

```diff
@@ -1945,6 +1948,9 @@ def _addBackendPlugin(self, backend_plugin: "BackendPlugin") -> None:
     def getBackendPlugins(self) -> List["BackendPlugin"]:
         return self._backend_plugins
 
+    def _addViewManipulator(self, view_manipulator: "PluginObject"):
```

Comment:
```suggestion
    def _addViewManipulator(self, view_manipulator: "PluginObject") -> None:
```

If you don't define what is returned, it doesn't automatically assume it's none (it assumes it can be anything)